Kudu.Core/SiteExtensions/SiteExtensionManagerV2.cs

```diff
@@ -610,11 +610,11 @@ public async Task<bool> UninstallExtension(string id)
         {
             ITracer tracer = _traceFactory.GetTracer();
 
-            string installationDirectory = GetInstallationDirectory(id);
-
             SiteExtensionInfo info = await GetLocalExtension(id, checkLatest: false);
 
-            if (info == null || !FileSystemHelpers.DirectoryExists(info.LocalPath))
+            string installationDirectory = GetInstallationDirectory(info.Id);
+
+            if (info == null || !FileSystemHelpers.DirectoryExists(info.LocalPath) || installationDirectory == null)
             {
                 tracer.TraceError("Site extension {0} not found.", id);
```

Comment:
Looks great, could we add more context on the Trace Error to help? E.G tracer.TraceError("Site extension {0} not found during uninstall.", id);